src/415D90.c

```diff
@@ -3460,7 +3460,7 @@ void btl_state_update_player_menu(void) {
         case BTL_SUBSTATE_PLAYER_MENU_CHANGE_MEMBER_1:
             entryIdx = 0;
             for (i = 1; i < ARRAY_COUNT(playerData->partners); i++) {
-                s32 partnerId = D_8008EEC0[i];
+                s32 partnerId = PartnerIDFromMenuIndex[i];
```

Comment:
I'm still not sure about the capital "D" in ID. See https://stackoverflow.com/questions/15526107/acronyms-in-camelcase for discussion